packages-web/fieldset/src/__tests__/Fieldset.spec.tsx

```diff
@@ -0,0 +1,43 @@
+import { shallow } from "enzyme";
+import { shallowToJson } from "enzyme-to-json";
+import { createElement, Fragment, ReactNode } from "react";
+
+import { Fieldset, FieldsetProps } from "../components/Fieldset";
+
+describe("Fieldset", () => {
+    const defaultFieldsetProps: FieldsetProps = {
+        name: "fieldset",
+        className: "className",
+        style: {},
+        tabIndex: 0,
+        legend: "legend"
+    };
+
+    const defaultChildren: ReactNode = (
+        <Fragment>
+            <label>Name:</label>
+            <br />
+            <input type="text" name="employee_name" id="employee_name" />
+        </Fragment>
+    );
+
+    it("renders children and legend", () => {
+        const fieldset = shallow(<Fieldset {...defaultFieldsetProps}>{defaultChildren}</Fieldset>);
+
+        expect(shallowToJson(fieldset)).toMatchSnapshot();
+    });
+    it("renders only children when no legend is passed", () => {
+        const fieldset = shallow(
+            <Fieldset {...defaultFieldsetProps} legend={undefined}>
+                {defaultChildren}
+            </Fieldset>
+        );
+
+        expect(shallowToJson(fieldset)).toMatchSnapshot();
+    });
+    it("renders nothing when no children are passed", () => {
+        const fieldset = shallow(<Fieldset {...defaultFieldsetProps} />);
+
+        expect(shallowToJson(fieldset)).toMatchSnapshot();
```

Comment:
What happens if you use `expect(fieldset).toMatchSnapshot();` instead?

Reply:
Your suggestion works as well :) This is due to the following jest config
```
snapshotSerializers: ["enzyme-to-json/serializer"]
```

Reply:
Changed it.